packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
-## v3.0.0-dev
+## v1.0.0 (June 15, 2023)
 
-- Upgrade React v18, React DOM v18, React-hook-form v7, and Chakra 2 libraries [#1166](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1166)
-- Add app `above-header` and product-list `above-page-header` convenience components [#1183](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1183)
-- Remove `cross-fetch` dependency [#1160](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1160)
+-   First public release of package on npm under `@salesforce/retail-react-app`
+-   Upgrade React v18, React DOM v18, React-hook-form v7, and Chakra 2 libraries [#1166](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1166)
+-   Add app `above-header` and product-list `above-page-header` convenience components [#1183](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1183)
+-   Remove `cross-fetch` dependency [#1160](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1160)
+-   Migration to "hooks" / `@salesforce/commerce-sdk-react` integration follow the [upgrade guide](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/upgrade-to-v3.html) and [view the diff](https://github.com/SalesforceCommerceCloud/pwa-kit/compare/release-2.7.x...release-3.0.x?diff=unified#files_bucket)
 
 ## v2.7.1 (May 11, 2023)
```

Comment:
Similar https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1272#discussion_r1231233752. 

@salesforce/retail-react-app@2.7.1 doesn't exist. Do we want to remove those versions?

Reply:
I think we should still keep the older versions in there. But since a note has been added for those older versions, I think that is sufficient and we can keep the older versions.